feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -495,6 +504,18 @@ static void vif_config_custom_opt_set(struct blob_buf *b,
 			blobmsg_add_string(b, NULL, operator_name);
 			blobmsg_close_array(b, n);
 		}
+		else if (strcmp(opt, "dynamic_vlan") == 0) {
+			blobmsg_add_string(b, "dynamic_vlan", value);
+
+			if (strcmp(value, "1") == 0 || strcmp(value, "2") == 0) {
+				strncpy(value, "/etc/hostapd.vlan", 20);
```

Comment:
Does file /etc/hostapd.vlan exist already or should something be in this commit to create it?  I don't see it on my AP.

Reply:
Its created by hostapd